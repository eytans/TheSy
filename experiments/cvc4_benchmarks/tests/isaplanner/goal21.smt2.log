19:21:13 [WARN] Collecting statistics
19:21:13 [WARN]  ex is: zero
19:21:13 [WARN]  ex is: (succ zero)
19:21:13 [WARN]  ex is: (succ (succ zero))
19:21:13 [WARN] Running exploration without proof mode, but goals were given
19:21:13 [WARN] Proof mode: true
19:21:13 [WARN]  ex is: zero
19:21:13 [WARN]  ex is: (succ zero)
19:21:13 [WARN]  ex is: (succ (succ zero))
19:21:13 [WARN] Running exploration without proof mode, but goals were given
19:21:13 [WARN] Proof mode: true
19:21:13 [WARN] proved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:21:13 [WARN] proved: (leq zero ?ts_ph_Nat_0) => true
19:21:13 [WARN] proved: (less ?ts_ph_Nat_0 ?ts_ph_Nat_0) => false
19:21:20 [WARN] proved: (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1))
19:21:20 [WARN] proved: (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1)) => (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:21:20 [WARN] proved: (less (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0) => false
19:21:20 [WARN] proved: (less (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_0) => false
19:21:20 [WARN] proved: (less ?ts_ph_Nat_0 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (less zero ?ts_ph_Nat_1)
19:21:20 [WARN] Found all lemmas
