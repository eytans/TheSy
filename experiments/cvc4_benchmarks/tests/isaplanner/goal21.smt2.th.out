19:21:13 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(less ?x zero) => false
(less zero (succ ?x)) => true
(less (succ ?x) (succ ?y)) => (less ?x ?y)
(less ?x ?y) => (less (succ ?x) (succ ?y))
(plus zero ?n) => ?n
(plus (succ ?n) ?m) => (succ (plus ?n ?m))
(succ (plus ?n ?m)) => (plus (succ ?n) ?m)
Searcher: (less ?autovar5 (succ ?autovar1))
Searcher: (less ?autovar5 (succ ?autovar3))
Searcher: (less zero ?autovar6)
Searcher: (less (succ ?autovar4) ?autovar6)
19:21:13 [0m[33m[WARN] [0m ex is: zero
19:21:13 [0m[33m[WARN] [0m ex is: (succ zero)
19:21:13 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:21:13 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:21:13 [0m[33m[WARN] [0mProof mode: true
19:21:13 [0m[33m[WARN] [0m ex is: zero
19:21:13 [0m[33m[WARN] [0m ex is: (succ zero)
19:21:13 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:21:13 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:21:13 [0m[33m[WARN] [0mProof mode: true
19:21:13 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:21:13 [0m[33m[WARN] [0mproved: (leq zero ?ts_ph_Nat_0) => true
19:21:13 [0m[33m[WARN] [0mproved: (less ?ts_ph_Nat_0 ?ts_ph_Nat_0) => false
19:21:20 [0m[33m[WARN] [0mproved: (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1))
19:21:20 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1)) => (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:21:20 [0m[33m[WARN] [0mproved: (less (succ ?ts_ph_Nat_0) ?ts_ph_Nat_0) => false
19:21:20 [0m[33m[WARN] [0mproved: (less (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_0) => false
19:21:20 [0m[33m[WARN] [0mproved: (less ?ts_ph_Nat_0 (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (less zero ?ts_ph_Nat_1)
19:21:20 [0m[33m[WARN] [0mFound all lemmas
done in 7644
